### ISSUE TITLE
TypeError when runring git removed-branches

### ISSUE THREAD

**Issue by charleslowell-valstro** [nemisj/git-removed-branches#15]
```
% git removed-branches
TypeError: upParts is not iterable
    at /home/cowboyd/.volta/tools/image/packages/git-removed-branches/lib/node_modules/git-removed-branches/lib/find-stale.js:56:41
    at Array.forEach (<anonymous>)
    at module.exports.findLocalBranches (/home/cowboyd/.volta/tools/image/packages/git-removed-branches/lib/node_modules/git-removed-branches/lib/find-stale.js:45:11)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async module.exports.run (/home/cowboyd/.volta/tools/image/packages/git-removed-branches/lib/node_modules/git-removed-branches/lib/find-stale.js:30:5)
    at async /home/cowboyd/.volta/tools/image/packages/git-removed-branches/lib/node_modules/git-removed-branches/index.js:47:5
```

**Comment by ewjoachim** [comment]
Had the same issue, `npx git-removed-branches@'2.3.1-alpha.0'` worked.

**Comment by nemisj** [comment]
@ewjoachim that's nice. before i do final release, could you please send me the output log `git branch --format="%(refname:short)@{%(upstream)}"` so that i can understand, what could cause this behaviour?

**Comment by ewjoachim** [comment]
The repo wasn't public and there really was too much private info in branch names to post it, sorry. If you tell me what you're looking for, though, I can maybe check.

Note that now that the command worked, there's no more branches to check :)

That said, I did run the command before and while I don't have the output saved, I remember that the first listed branch had no upstream (`@{}`)

**Comment by nemisj** [comment]
@ewjoachim, i'm looking for the case which breaks the code ( and unfortunately it's not `@{}` ). it looks like the upstream might have some other prefix than '/refs/remotes/', while it's pretty odd and I can't find any information on that in the documentation. The output, even without real branch names, would allow me to understand, what this 'other prefix' is

**Comment by nemisj** [comment]
Fixed in 2.3.1
